src/utils/sdfg/sdfg_range_elem_to_string.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019-2021 ETH Zurich and the DaCe authors. All rights reserved.
```

Comment:
This file should be merged together with sdfg_property_to_string.js (maybe call them to_string.js or display.js?)